packages/pelagos/src/components/TagInput.js

```diff
@@ -55,20 +57,22 @@ const TagInput = ({
 
 	const addTag = useCallback(
 		(name) => {
-			if (!hasTag(tags, name)) {
-				const error = validate(name);
+			const transformed = transform(name);
+
+			if (!hasTag(tags, transformed)) {
+				const error = validate(transformed);
 				if (error) {
 					onError(error);
 				} else {
-					liveRef.current.textContent = t`${name} added`;
-					onChange([...tags, name]);
+					liveRef.current.textContent = t`${transformed} added`;
```

Comment:
Unfortunately this changes the translation, please change the parameter to something else (maybe `inputName`?) and change `transformed` back to `name`